lms/djangoapps/verify_student/views.py

```diff
@@ -699,6 +699,176 @@ def _response_if_deadline_passed(self, course, deadline_name, deadline_datetime)
             return render_to_response("verify_student/missed_deadline.html", context)
 
 
+class SubscriptionView(View):
+
+    @method_decorator(login_required)
+    def get(self, request):
+        """
+        Render the payment flow.
+
+        Arguments:
+            request (HttpRequest): The request object.
+
+        Returns:
+            HttpResponse
+
+        Raises:
+            Http404: The course does not exist.
+        """
+        # Get the course key
+        course_key = CourseKey.from_string(settings.SUBSCRIPTION_COURSE_KEY)
```

Comment:
Just trying to understand :) what is the `SUBSCRIPTION_COURSE_KEY` ? some special course we create to test if users have access to it?